CodeEditModules/Modules/AppPreferences/src/Sections/ThemePreferences/Model/ThemeModel.swift

```diff
@@ -153,7 +153,9 @@ public final class ThemeModel: ObservableObject {
             "codeedit-xcode-dark",
             "codeedit-xcode-light",
             "codeedit-github-dark",
-            "codeedit-github-light"
+            "codeedit-github-light",
+            "codeedit-midnight"
+            
```

Comment:
```suggestion

```

SwiftLint is complaining about trailing whitespace.
To fix it, the tabs have to be removed.